templates/prSignUp.html

```diff
@@ -13,14 +13,14 @@
 
 
     <!-- import the webpage's stylesheet -->
-    <link rel="stylesheet" href="/prSignUp.css" />
-    <link rel="stylesheet" type="text/css" href="temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" href="../assets/css/prSignUp.css" />
```

Comment:
Use the direct names of directories and subdirectories instead of using ".." (the relative file pathways). Try and change the ".." to be specific throughout the code.

Reply:
replace ".." file pathways w/ specific file names on all lines where it is present